reports/performance/_outliers.qmd

```diff
@@ -18,34 +18,38 @@ exclusively on these price-based factors that determine outlier status.
 ### Count by Type
 
 ```{r _outliers_type_breakdown}
-# Make new column for a few cells of the outliers section
+# Subset important outlier information
 training_data <- training_data %>%
-  mutate(price_outlier_reason = case_when(
-    str_detect(sv_outlier_reason1, regex("price", ignore_case = TRUE)) ~ sv_outlier_reason1,
-    str_detect(sv_outlier_reason2, regex("price", ignore_case = TRUE)) ~ sv_outlier_reason2,
-    str_detect(sv_outlier_reason3, regex("price", ignore_case = TRUE)) ~ sv_outlier_reason3,
-    TRUE ~ NA_character_
-  ))
+  mutate(
+    outlier_reasons_to_graph = if_else(
+      sv_is_outlier == TRUE,
+      paste0(sv_outlier_reason1, " (", sv_outlier_reason2, ")"),
+      NA_character_
+    )
+  ) %>%
+  # Yank NA string values to make output cleaner
+  mutate(outlier_reasons_to_graph = str_remove_all(outlier_reasons_to_graph, " \\(NA\\)"))
```

Comment:
**suggestion:** Maybe cleaner to do this in the `if_else` above. Convert it to a `case_when()` and return only the first reason if the second reason is NA, etc.